cz.zeropage.Formiko.json

```diff
@@ -31,253 +35,43 @@
-                    "url": "https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl",
-                    "sha256": "3350809f0555b11f552448330d0b52d5f24c91a322ea4a15ef22629740f3761c"
-                },
                 {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/e4/2c/0cde8bc14780f8a0f7f2466444088650016d0194370a9a68f161bd8db2d7/docutils_html5_writer-1.0.3-py3-none-any.whl",
-                    "sha256": "5bc8a39a148eca414774acd8e44f0708954376c84bfc1a35ac43fe5cdbf06ff1"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/a5/62/bbd2be0e7943ec8504b517e62bab011b4946e1258842bc159e5dfde15b96/html5lib-1.0.1-py2.py3-none-any.whl",
-                    "sha256": "20b159aa3badc9d5ee8f5c647e5efd02ed2a66ab8d354930bd9ff139fc1dc0a3"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/22/cd/a6aa959dca619918ccb55023b4cb151949c64d4d5d55b3f4ffd7eee0c6e8/docutils-0.15.2-py3-none-any.whl",
-                    "sha256": "6c4f696463b79f1fb8ba0c594b63840ebd41f059e92b31957c46b74a4599b6d0"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/c4/43/3f1e7d742e2a7925be180b6af5e0f67d38de2f37560365ac1a0b9a04c015/lxml-4.4.1.tar.gz",
-                    "sha256": "c81cb40bff373ab7a7446d6bbca0190bccc5be3448b47b51d729e37799bb5692"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/f4/24/2a3e3df732393fed8b3ebf2ec078f05546de641fe1b667ee316ec1dcf3b7/webencodings-0.5.1-py2.py3-none-any.whl",
-                    "sha256": "a0af1213f3c2226497a97e2b3aa01a7e4bee4f403f95be16fc9acd2947514a78"
-                },
-                {
-                    "type": "file",
-                    "url": "https://files.pythonhosted.org/packages/d4/70/d60450c3dd48ef87586924207ae8907090de0b306af2bce5d134d78615cb/python_dateutil-2.8.1-py2.py3-none-any.whl",
-                    "sha256": "75bb3f31ea686f1197762692a9ee6a7550b59fc6ca3a1f4b5d7e32fb98e2da2a"
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/gtkspell/gtkspell3-3.0.10.tar.xz",
+                    "sha256": "b040f63836b347eb344f5542443dc254621805072f7141d49c067ecb5a375732"
                 }
             ]
         },
+        "python3-modules.json",
         {
             "name": "formiko",
             "buildsystem": "simple",
             "build-commands": [
-                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} formiko"
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --root=/ --optimize=1"
```

Comment:
Why is installing from source code (git) via setup.py better, than installing via pip from python wheel?

Reply:
I did it mostly to differentiate between the python dependencies and the actual app being packaged; and it's also a personal preference. Also it'd be easier to apply small patches quickly if the build somehow breaks (broken appdata/icon issue etc.) during runtime/dependency bumps than waiting for a release.